File: backends/fiber/internal/models/comment.go
package models

import (
	"time"

	"github.com/google/uuid"
)

type Comment struct {
	ID          uuid.UUID `gorm:"column:comment_id;primaryKey;default:gen_random_uuid()" json:"id"`
	RecipeID    uuid.UUID `gorm:"column:recipe_id" json:"-"`
	UserID      uuid.UUID `gorm:"column:user_id" json:"-"`
	User        User      `json:"user,omitempty"`
	CommentText string    `gorm:"column:comment_text;not null" json:"comment_text"`
	CreatedAt   time.Time `gorm:"column:created_at;default:CURRENT_TIMESTAMP" json:"created_at"`
	UpdatedAt   time.Time `gorm:"column:updated_at;default:CURRENT_TIMESTAMP" json:"updated_at"`
}

File: backends/aspnet/Recipes.API/entities/Comment.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Recipes.API.entities;

public partial class Comment
{
    [JsonPropertyName("id")]
    public Guid CommentId { get; set; }

    public Guid? RecipeId { get; set; }

    public Guid? UserId { get; set; }

    public string CommentText { get; set; } = null!;

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public virtual Recipe? Recipe { get; set; }

    public virtual User? User { get; set; }
}
